docs/user-guide/quickstarts/generic.md

```diff
@@ -0,0 +1,99 @@
+# Getting Started with Screwdriver (Generic Quickstart)
+
+This page will cover how to build and deploy a sample app with Screwdriver in minutes. In this example, we are using the SCM provider Github.
+
+## Requirements
+- Github account
+
+## Set up
+First, fork and clone the [sample Generic app](https://github.com/screwdriver-cd-test/quickstart-generic) into your local development environment and cd into the project directory.
+
+```bash
+$ cd quickstart-generic/
```

Comment:
Gotcha. Maybe put a placeholder in there, like `git clone git@github.com:<YOUR_USERNAME_HERE>/quickstart-generic.git` ?